diagram.md

```diff
@@ -141,7 +143,7 @@ The `Tracks Elements` tells us the technical details of what is in each track. F
 ```
 Figure: Representation of the `Tracks Element` and a selection of its `Descendant Elements`.
 
-The `Chapters Element` section lists all of the Chapters. Chapters are a way to set predefined points to jump to in video or audio.
+The `Chapters Element` lists all of the Chapters. Chapters are a way to set predefined points to jump to in video or audio.
```

Comment:
I'd downcase the "Chapters" at the end of the sentence as you're not talking about the element itself but about the concept.